core/src/main/java/com/google/cloud/sql/RefreshStrategy.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.sql;
+
+/** Enum for supported refresh strategies. */
+public enum RefreshStrategy {
+  BACKGROUND,
+  LAZY
```

Comment:
Since Lazy doesn't do anything yet, shall we leave this out and introduce it again when it actually triggers a change?